CHANGELOG.md

```diff
@@ -55,6 +55,7 @@
 * [ENHANCEMENT] Distributor: add support for ingesting exponential histograms that are over the native histogram scale limit of 8 in OpenTelemetry format by downscaling them. #5532
 * [ENHANCEMENT] General: buffered logging: #5506
   * `-log.buffered`: Enable buffered logging
+* [ENHANCEMENT] Distributor Ingester: Add metrics to count the number of requests rejected for hitting per-instance limits, `cortex_distributor_instance_discarded_requests_total` and `cortex_ingester_instance_discarded_requests_total` respectively. #5551
```

Comment:
We use "discarded" in other context already -- requests for which we return 4xx errors due to various limits.

These requests are rejected with 5xx error instead, and we expect that they will be repeated by client. I'd suggest using different term than "discarded" (eg. rejected, as you use in comments)

Reply:
Renamed.